client/workflows/client.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package workflows
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/mendersoftware/go-lib-micro/identity"
+	"github.com/mendersoftware/go-lib-micro/requestid"
+	"github.com/mendersoftware/go-lib-micro/rest.utils"
+	"github.com/pkg/errors"
+)
+
+const (
+	HealthCheckURI = "/api/v1/health"
+	ResetEmailURI  = "/api/v1/workflow/send_password_reset_email"
```

Comment:
do we need this in the deviceconfig?

Reply:
ops, sorry, copy-and-paste from another package. Removed!